multichain-testing/test/account-balance-queries.test.ts

```diff
@@ -67,7 +67,8 @@ const queryAccountBalances = test.macro({
     });
 
     const offerResult = await retryUntilCondition(
-      () => vstorageClient.queryData(`published.wallet.${agoricAddr}`),
+      () => smartWalletKit.readPublished(`wallet.${agoricAddr}`),
+      // @ts-expect-error UpdateRecord may not have 'status'
       ({ status }) => status.id === offerId && (status.result || status.error),
```

Comment:
if there's no status, `status.id` will throw, right?

```suggestion
      (u) => u?.status.id === offerId && (u?.status.result || u?.status.error),
```

Reply:
Have you considered the [seatLike](https://github.com/Agoric/agoric-sdk/blob/master/multichain-testing/tools/e2e-tools.js#L595) `.getOfferResult()` API?

Reply:
haven't considered that yet. I was just trying to DRY out what I could.

do you think that API should be part of client-utils?